base/linalg/bidiag.jl

```diff
@@ -40,6 +40,15 @@ function getindex{T}(A::Bidiagonal{T}, i::Integer, j::Integer)
     i == j ? A.dv[i] : (A.isupper && (i == j-1)) || (!A.isupper && (i == j+1)) ? A.ev[min(i,j)] : zero(T)
 end
 
+## structured matrix methods ##
+function Base.replace_in_print_matrix(A::Bidiagonal,i::Integer,j::Integer,s::AbstractString)
+  if A.isupper
```

Comment:
Guess I hadn't read about that particular convention just yet. I'll make the changes by hand for now, but it would be nice to have something automatic. Is there an official linter that conforms to the style in the manual so I can write a plugin for my editor?